docs/proxy/datacenter_proxy/nodejs_examples.md

```diff
@@ -126,9 +128,15 @@ Use one IP chosen from `SHADER` and `BUYPROXIES94952` proxy groups for the brows
 
     Apify.main(async () => {
         const password = process.env.APIFY_PROXY_PASSWORD;
-        const username = 'groups-SHADER+BUYPROXIES94952,session-my_session';
+        const proxyConfiguration = await Apify.createProxyConfiguration({
+            password,
+            // if you need to use more then one group
+            // simply add the additional ID's to the array below
+            groups: ['SHADER', 'BUYPROXIES94952'],
```

Comment:
I don't think that we need this comment everywhere, when we have an example that deals with multiple groups.

Reply:
Removed.